backend/src/tests/e2e/users.test.ts

```diff
@@ -27,6 +27,16 @@ describe('POST /user', () => {
     expect(req.statusCode).toEqual(400);
   });
 
+  it('returns 400 when the password is less than 8 characters', async () => {
+    const req = await request(app).post('/user').send({
+      email: '',
+      username: 'admin',
+      password: 'pass',
+    });
+
+    expect(req.statusCode).toEqual(400);
```

Comment:
We have different outputs with 400 status code. Maybe it's interesting expect the message to validate that is the 'Password is too short' message.

Reply:
Done